apps/mobile/src/components/InsideRouter/atoms.ts

```diff
@@ -18,6 +27,19 @@ export const refreshBtcPriceActionAtom = atom<undefined, [], Task<boolean>>(
   (get, set) => {
     const api = get(privateApiAtom)
     const {t} = get(translationAtom)
+    const btcPriceLastUpdateAt = get(btcPriceLastUpdateAtAtom)
+    const timeDifferenceSinceLastUpdate = Math.round(
+      DateTime.now().diff(
+        DateTime.fromMillis(btcPriceLastUpdateAt ?? Date.now()),
```

Comment:
No need to convert to DateTime anymore. You can subtract milliseconds directly ;).

Also, remove `?? Date.now()` otherwise it won't trigger the update with initial value (the following condition will be truthy).